stdlib/internal/SwiftSyntax/SourcePresence.swift

```diff
@@ -0,0 +1,24 @@
+//===---------------- SourcePresence.swift - Source Presence --------------===//
+//
+// This source file is part of the Swift.org open source project
+//
+// Copyright (c) 2014 - 2017 Apple Inc. and the Swift project authors
+// Licensed under Apache License v2.0 with Runtime Library Exception
+//
+// See https://swift.org/LICENSE.txt for license information
+// See https://swift.org/CONTRIBUTORS.txt for the list of Swift project authors
+//
+//===----------------------------------------------------------------------===//
+
+import Foundation
+
+/// An indicator of whether a Syntax node was found or written in the source.
+///
+/// This does not mean, necessarily, that the source item is considered "implicit".
```

Comment:
Is this trying to say that `.missing` does not imply "implicit", but instead might be a required syntax element that was omitted?